apps/spruce/src/pages/preferences/preferencesTabs/NotificationsTab.tsx

```diff
@@ -49,7 +48,7 @@ export const NotificationsTab: React.FC = () => {
   });
 
   if (loading) {
-    return <Skeleton active />;
+    return <FormSkeleton />;
```

Comment:
it looks like we're using `FormSkeleton` a lot for modals but not for pages. Should this have a different skeleton?

Reply:
Swapped it with Card since thats what renders here